include/couch_mrview.hrl

```diff
@@ -94,7 +94,10 @@
     resp,
     prepend,
     etag,
-    should_close = false
+    should_close = false,
+    buffer = [],
+    bufsize = 0,
+    threshold = 1490
```

Comment:
And maybe even define it as a macro as you're using it in a few places; minor nit though.